quarkus/server/distribution/README.md

```diff
@@ -0,0 +1,36 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+ 
+   http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Apache Polaris - Server
+
+## Prerequisites
+
+Polaris server requires a Java SE 21 or higher to run.
+
+## Start the server
+
+In a command line, in the directory where you extracted the Polaris server archive, run the following command to start the server:
+
+```
+./run.sh
+```
```

Comment:
It would be nice if we talked briefly about the root credential configuration. Maybe we just need to link this page, https://polaris.apache.org/in-dev/unreleased/configuring-polaris-for-production/. It's not a blocker. We can improve this later.

Reply:
I added link to documentation in admin and server `README.md`.